### PR TITLE
tests: add longtests

### PR DESCRIPTION
**Pull request by jorainer**
- Move unit tests for the *old* xcms classes to the longtests folder.

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ local_data
 README.md
 Rplots.pdf
 .DS_Store
+.BBSoptions
```
Comment (sneumann) [inline .Rbuildignore:11]: Why add to .gitignore ? These are control files for BioC build system, so it'd need to arrive there,

File: tests/testthat/test_do_adjustRtime-functions.R [hunk 2]
```diff
@@ -1,9 +1,9 @@
 test_that("getPeakGroupsRtMatrix works", {
     param <- PeakGroupsParam()
     nSamples <- length(fileNames(xod_xg))
-    pkGrp <- .getPeakGroupsRtMatrix(
+    pkGrp <- xcms:::.getPeakGroupsRtMatrix(
```
Comment (sneumann) [inline tests/testthat/test_do_adjustRtime-functions.R:4]: So all non-exported functions in tests need to be prefixed ?

File: .Rbuildignore (hunk 1 above)
Comment (jorainer) [reply .Rbuildignore:11]: I was not sure - `R CMD check` complains about this file. But you're right, I will remove that line again.

File: tests/testthat/test_do_adjustRtime-functions.R (hunk 2 above)
Comment (jorainer) [reply tests/testthat/test_do_adjustRtime-functions.R:4]: No - that's a leftover when I run the tests line by line locally.

File: .Rbuildignore (hunk 1 above)
Comment (jorainer) [reply .Rbuildignore]: I removed this in the latest commit.

**Comment by jorainer** [comment]
> Can we have instructions on longtests / linkt to the BioC page somewhere

what exactly do you mean? Once the first *long test* was performed we could link that page in the README.md.

**Comment by sneumann** [comment]
lgtm. Yours, Steffen

**Pull request merged**